ghcup-0.0.7.yaml

```diff
@@ -3883,13 +3883,128 @@ ghcupDownloads:
+              dlOutput: ghc-9.8.1-x86_64-linux-ubuntu18_04.tar.xz
+              dlSubdir: ghc-9.8.1-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.1/ghc-9.8.1-x86_64-ubuntu18_04-linux.tar.xz
+            '>= 20': &ghc981-x86_64-ubuntu20_04
+              dlHash: 436a34dffafdd0fe2019e973805d479b6a0494f7bd1200502efa95a3c73053b1
+              dlOutput: ghc-9.8.1-x86_64-linux-ubuntu20_04.tar.xz
+              dlSubdir: ghc-9.8.1-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.1/ghc-9.8.1-x86_64-ubuntu20_04-linux.tar.xz
+          Linux_RedHat:
+            unknown_versioning: *ghc981-x86_64-centos7
+          Linux_Ubuntu:
+            ( >= 16 && < 19 ): *ghc981-x86_64-ubuntu18_04
+            unknown_versioning: *ghc981-x86_64-ubuntu20_04
+          Linux_UnknownLinux:
+            unknown_versioning:
+              dlHash: 3259df634fc8de876224007400ca2eec73831c59a9e55687809b66553a73c5f5
+              dlOutput: ghc-9.8.1-x86_64-linux-rocky8.tar.xz
+              dlSubdir: ghc-9.8.1-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.1/ghc-9.8.1-x86_64-rocky8-linux.tar.xz
+          Windows:
+            unknown_versioning:
+              dlHash: eb74654419cb646e9d2e362b57f2f81bc9f5231bceb543b14cb0341aedb07fa3
+              dlOutput: ghc-9.8.1-x86_64-windows.tar.xz
+              dlSubdir: ghc-9.8.1-x86_64-unknown-mingw32
+              dlUri: https://downloads.haskell.org/~ghc/9.8.1/ghc-9.8.1-x86_64-unknown-mingw32.tar.xz
+        A_ARM64:
+          Darwin:
+            unknown_versioning:
+              dlHash: a38bb06f60c92d348e3ee927a4a88e88488f874e99f440d06244a4e4f9db9e3b
+              dlOutput: ghc-9.8.1-aarch64-darwin.tar.xz
+              dlSubdir: ghc-9.8.1-aarch64-apple-darwin
+              dlUri: https://downloads.haskell.org/~ghc/9.8.1/ghc-9.8.1-aarch64-apple-darwin.tar.xz
+          Linux_UnknownLinux:
+            unknown_versioning:
+              dlHash: aab7af72614f8bf9ca624407aa4dbc69bc009c2b4cc1a0f3c062008db81bdb95
+              dlOutput: ghc-9.8.1-aarch64-linux-deb10.tar.xz
+              dlSubdir: ghc-9.8.1-aarch64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.1/ghc-9.8.1-aarch64-deb10-linux.tar.xz
+      viChangeLog: https://downloads.haskell.org/~ghc/9.8.1/docs/users_guide/9.8.1-notes.html
+      viReleaseDay: '2023-10-09'
+      viSourceDL:
+        dlHash: b2f8ed6b7f733797a92436f4ff6e088a520913149c9a9be90465b40ad1f20751
+        dlOutput: ghc-9.8.1-src.tar.xz
+        dlSubdir: ghc-9.8.1
+        dlUri: https://downloads.haskell.org/~ghc/9.8.1/ghc-9.8.1-src.tar.xz
+      viTags:
+      - Latest
+      - base-4.19.0.0
+      viTestDL:
+        dlHash: 2626f3bb974428cbb795c5433017e18088f0d8de8a66d5a23ff4776847a5bec3
+        dlOutput: ghc-9.8.1-testsuite.tar.xz
+        dlSubdir: ghc-9.8.1
+        dlUri: https://downloads.haskell.org/~ghc/9.8.1/ghc-9.8.1-src.tar.xz
```

Comment:
@bgamari this is garbage... what is this?

Reply:
https://gitlab.haskell.org/ghc/ghc/-/issues/24268